tests/integration/api/v1/recovery/post.test.js

```diff
@@ -129,21 +131,21 @@ describe('POST /api/v1/recovery', () => {
 
       const responseBody = await response.json();
 
-      expect(response.status).toEqual(400);
-
+      expect(response.status).toBe(201);
       expect(responseBody).toStrictEqual({
-        name: 'ValidationError',
-        message: 'O "email" informado não foi encontrado no sistema.',
-        action: 'Ajuste os dados enviados e tente novamente.',
-        status_code: 400,
-        error_id: responseBody.error_id,
-        request_id: responseBody.request_id,
-        error_location_code: 'MODEL:RECOVERY:FIND_USER_BY_USERNAME_OR_EMAIL:NOT_FOUND',
-        key: 'email',
+        used: false,
+        expires_at: responseBody.expires_at,
+        created_at: responseBody.created_at,
+        updated_at: responseBody.updated_at,
       });
 
-      expect(uuidVersion(responseBody.error_id)).toEqual(4);
-      expect(uuidVersion(responseBody.request_id)).toEqual(4);
+      expect(Date.parse(responseBody.expires_at)).not.toEqual(NaN);
```

Comment:
Aqui poderia ser `toBe` no lugar de `toEqual`, mas existe algo ainda melhor, que é:

```suggestion
      expect(Date.parse(responseBody.expires_at)).not.toBeNaN();
```